### ISSUE TITLE
Wording activity stamp for confirmed transaction <1day

### ISSUE THREAD

**Issue by MaudSimon** [alephium/alephium-frontend#262]
**Describe the bug**
While a transaction is pending the timestamp tracks accurately the timeframe "a sec ago, 1minute, 5 minutes ago" As soon as the transaction is confirmed the timestamp changes to  "in a day". 

I'm not sure if it is intended ? Even if it is, the sentence seems wrongly turned shouldn't it be "less than a day ago" ?

<img width="267" alt="image" src="https://github.com/alephium/alephium-frontend/assets/52083111/da8b49df-32b8-4c51-baf5-af6e445e0601">

**To Reproduce**
Check transaction history in wallet/explorer for confirmed transaction dating less than a day. 

**Expected behavior**
Either the "timestamp" of confirmed transactions should say, "x minutes/hours ago" up to a day or the message should be edited to "less than a day ago".

**Comment by nop33** [comment]
Yes, it should always say "ago", we can't have future transactions. Thanks for the report.

**Comment by nit27** [comment]
I also have a related issue on MW. When new txs are done for **a day**, the timestamps still say `a few seconds ago`. Try to scroll down to re-fetch the data sometimes help or kill app and re-open. I belive we should have a 🔁 button.
